### PR TITLE
CHAOS-2466: Add CE SMP roadmap and k8s fault permissions 

### PR DESCRIPTION
**Pull request by sagarkrsd**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot. 

### PREVIEW LINKS:

[whats-supported](https://64f737d263929d0afc48dba9--harness-developer.netlify.app/docs/chaos-engineering/get-started/whats-supported)
[fault-container-permissions](https://64f737d263929d0afc48dba9--harness-developer.netlify.app/docs/chaos-engineering/technical-reference/chaos-faults/kubernetes/security-considerations/container-permissions)
[ce-on-harness-self-managed-enterprise-edition](https://64f737d263929d0afc48dba9--harness-developer.netlify.app/docs/category/ce-on-harness-self-managed-enterprise-edition)

### CODE REVIEW

**Review by ajeshbaby — DISMISSED** [review]
LGTM

**Comment by sagarkrsd** [comment]
@brian-f-harness Can you please help with the review of SMP related sections in this PR, thanks!

File: docs/chaos-engineering/get-started/whats-supported.md [hunk 1]
```diff
@@ -3,25 +3,33 @@ title: What's supported
 sidebar_position: 10
 ---
 
-This topic lists the platforms and technologies that Harness Chaos Engineering (CE) supports.
+This topic lists the platforms and technologies that Harness Chaos Engineering (CE) supports for deploying chaos infrastructure.
+To learn more about chaos infrastructure, please refer [chaos infrastructure](/docs/chaos-engineering/technical-reference/architecture#chaos-infrastructure).
```
Comment (brian-f-harness) [inline docs/chaos-engineering/get-started/whats-supported.md:7]:
```suggestion
To learn more about chaos infrastructure, go to [Chaos infrastructure](/docs/chaos-engineering/technical-reference/architecture#chaos-infrastructure).
```

File: docs/chaos-engineering/get-started/whats-supported.md [hunk 2]
```diff
@@ -3,25 +3,33 @@ title: What's supported
...
+
 
 ## Chaos infrastructure
 
-You can deploy chaos infrastructure on the platforms below.
+### Supported operating systems and distributions
+
+The following table shows the supported operating systems and distributions for deploying chaos infrastructure.
+
+| Platform | Distribution | Version | Harness cloud | Self-Managed enterprise edition |
```
Comment (brian-f-harness) [inline docs/chaos-engineering/get-started/whats-supported.md:16]:
```suggestion
| Platform | Distribution | Version | Harness cloud | Self-Managed Enterprise Edition |
```

File: docs/chaos-engineering/get-started/whats-supported.md [hunk 3]
```diff
@@ -56,3 +51,12 @@ The following table shows available fault types and where they are supported. Fo
 | **Resource stress**<br />For example, CPU stress, Memory stress, etc. | ✅| ✅  | ✅     |  ✅    |  ✅  |    |  ✅  |
 | **State change**<br />For example, VM stop, Pod kill, etc.  |  ✅  |  ✅  |   ✅   |   ✅   |  ✅  |   ✅  |   ✅   |
 | **Managed Services**<br />For example, AWS ECS agent stop, Azure web app stop, etc. |   |   |   |   ✅   |  ✅  |   ✅  |   ✅   |
+
+## CE on Harness Self-Managed Enterprise Edition (SMP)
```
Comment (brian-f-harness) [inline docs/chaos-engineering/get-started/whats-supported.md:55]:
```suggestion
## CE on Harness Self-Managed Enterprise Edition
```

File: docs/chaos-engineering/get-started/whats-supported.md [hunk 4]
```diff
@@ -56,3 +51,12 @@ The following table shows available fault types and where they are supported. Fo
...
+
+Chaos Engineering on Harness Self-Managed Enterprise Edition is in feature parity with Harness Cloud(SaaS) with the following limitations:
+* Harness enterprise chaos hub is not supported by default.
+* Please contact [Harness Support](mailto:support@harness.io) for getting access to Harness enterprise chaos hub.
```
Comment (brian-f-harness) [inline docs/chaos-engineering/get-started/whats-supported.md:59]:
```suggestion
* Contact [Harness Support](mailto:support@harness.io) for access to the Harness enterprise chaos hub.
```

File: docs/chaos-engineering/get-started/whats-supported.md [hunk 5]
```diff
@@ -56,3 +51,12 @@ The following table shows available fault types and where they are supported. Fo
...
+* Once you have access to enterprise chaos hub, you can add a custom chaos hub by following the steps mentioned [here](/docs/chaos-engineering/configure-chaos-experiments/chaos-hubs/add-chaos-hub).
```
Comment (brian-f-harness) [inline docs/chaos-engineering/get-started/whats-supported.md:60]:
```suggestion
* Once you have access to the enterprise chaos hub, you can add a custom chaos hub. For more information, go to [Add a custom chaos hub](/docs/chaos-engineering/configure-chaos-experiments/chaos-hubs/add-chaos-hub).
```

**Review by brian-f-harness — COMMENTED** [review]
@sagarkrsd I made a couple of suggestions, mostly for docs standards. Assigned to Prateek to review SMP details.

**Comment by brian-f-harness** [comment]
@pratmit I made docs standards suggestions, but am unable to approve for overall SMP content.

File: docs/chaos-engineering/get-started/whats-supported.md [hunk 6]
```diff
@@ -3,25 +3,33 @@ title: What's supported
...
+
 
 ## Chaos infrastructure
 
-You can deploy chaos infrastructure on the platforms below.
+### Supported operating systems and distributions
+
+The following table shows the supported operating systems and distributions for deploying chaos infrastructure.
+
+| Platform | Distribution | Version | Harness cloud | Self-Managed Enterprise Edition |
```
Comment (brian-f-harness) [inline docs/chaos-engineering/get-started/whats-supported.md:16]:
```suggestion
| Platform | Distribution | Version | Harness SaaS | Self-Managed Enterprise Edition |
```

File: docs/chaos-engineering/technical-reference/chaos-faults/kubernetes/security-considerations/container-permissions.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+title: Fault container permissions
+sidebar_position: 10
+---
+
+This topic lists the various categories of Kubernetes faults and the container permissions required to execute them.
+
+
+### Supported Kubernetes faults and required container permissions
+
+The following table shows the supported Kubernetes faults and the permissions required on fault containers to execute them.
```
Comment (brian-f-harness) [inline docs/chaos-engineering/technical-reference/chaos-faults/kubernetes/security-considerations/container-permissions.md:11]:
```suggestion
The following table shows the supported Kubernetes faults and the required fault container permissions to execute them.
```

**Review by brian-f-harness — APPROVED** [review]

**Pull request merged**